frontend/components/initiative/Editor.vue

```diff
@@ -0,0 +1,119 @@
+<script setup lang="ts">
+import { type EditorInitiative, isComplete } from '@/lib/editor'
+
+const props = defineProps<{
+  editorInitiative: EditorInitiative
+}>()
+
+const emit = defineEmits<(e: 'update:editorInitiative', ei: EditorInitiative) => void>()
+
+const model = computed({
+  get: () => props.editorInitiative,
+  set: (editorInitiative: EditorInitiative) => { emit('update:editorInitiative', editorInitiative) },
+})
+</script>
+
+<template>
```

Comment:
I ran this locally and it looks excellent, great work!

Reply:
Thanks!